Components/BasketHelper.php

```diff
@@ -645,7 +642,8 @@ public function findPercentagedIndividualVouchers($voucherCode)
             ->setParameter('voucherCode', $voucherCode);
 
 
-        return $builder->getQuery()->getResult();    }
+        return $builder->getQuery()->getResult();
```

Comment:
There is a statement which forces us to make a new line before every `return` statement but I cannot see any configuration for cases where a useless line was added... :/